pkg/Makefile

```diff
@@ -12,6 +12,8 @@ DESCRIPTION = "Aerospike Prometheus Exporter, An agent to export Aerospike metri
 LICENSE = "Apache License 2.0"
 URL = "https://github.com/aerospike/aerospike-prometheus-exporter"
 VENDOR = "Aerospike, Inc."
+ARCH ?= $(shell uname -m)
+DEB_PKG_ARCH ?= $(shell dpkg-architecture -q DEB_BUILD_ARCH)
```

Comment:
I think in some places you may get `aarch64` instead of `arm64`, so you may need to do something like this,
```
ifeq ($(ARCH),aarch64)
        override ARCH=arm64
endif
ifeq ($(ARCH),x86_64)
        override ARCH=amd64
endif
```

Reply:
I think x86_64 and aarch64 are valid architecutres/names for rpm packages. DEB_PKG_ARCH takes care of this for deb packages.

Reply:
Okay. We have been actually releasing it `x86_64` in rpm packages and `amd64` in deb packages. So that's alright. We can continue the same.